listenbrainz/webserver/views/feedback_api.py

```diff
@@ -180,9 +180,12 @@ def get_feedback_for_recordings_for_user(user_name):
 
     recordings = request.args.get('recordings')
 
+    if not recordings:
```

Comment:
You should also check if passed recording msids are valid uuids.

Reply:
Thats actually checked in pydantic model right now. e.g. https://api.listenbrainz.org/1/feedback/user/shivam-kapila/get-feedback-for-recordings?recordings=foo

Reply:
Please let me know if this needs change

Reply:
Ah cool!

Reply:
I will have a second pass tomorrow :)

Reply:
https://api.listenbrainz.org/1/feedback/user/Vansika%20Pareek/get-feedback-for-recordings?recording=foo returns 500?

Reply:
I am not really sure about the SQL query you are using here, but I think the model will check DB/SQL returned `results` and the not the values/args passed to the SQL query which are directly taken from the URL and may not be in the desired format. have a look :)

Reply:
I wrote `recording` in place of `recordings`. Nevamind. Looks good.

Reply:
This 500 will vanish after the PR is moved to prod. But the other one is still 400.

Reply:
Yeah, my bad. I missed the `s` in the spelling :p